components/style/servo_config_facade.rs

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ /* Ref: https://github.com/servo/servo/issues/15460 for more info for why it's needed. */
+
+// TODO: write docs for this mod
+use servo_config;
```

Comment:
rustc should give you an error like this when you run `./mach build-geckolib`:
```
error[E0432]: unresolved import `servo_config`
  --> /Users/canova/projects/servo/components/style/config.rs:11:5
   |
11 | use servo_config;
   |     ^^^^^^^^^^^^ no `servo_config` in the root

error: aborting due to previous error
```
What OS do you use? If this didn't show up, maybe it was a rustc bug?